apollo-portal/src/main/resources/static/scripts/directive/release-modal-directive.js

```diff
@@ -88,38 +88,38 @@ function releaseModalDirective($translate, toastr, AppUtil, EventManager, Releas
                     scope.toReleaseNamespace.releaseTitle,
                     scope.releaseComment,
                     scope.isEmergencyPublish).then(
-                        function (result) {
-                            AppUtil.hideModal('#releaseModal');
-                            toastr.success($translate.instant('ReleaseModal.GrayscalePublished'));
-
-                            scope.releaseBtnDisabled = false;
-
-                            //refresh item status
-                            scope.toReleaseNamespace.branchItems.forEach(function (item, index) {
-                                if (item.isDeleted) {
-                                    scope.toReleaseNamespace.branchItems.splice(index, 1);
-                                } else {
-                                    item.isModified = false;
-                                }
-                            });
-                            //reset namespace status
-                            scope.toReleaseNamespace.itemModifiedCnt = 0;
-                            scope.toReleaseNamespace.lockOwner = undefined;
-
-                            //check rules
-                            if (!scope.toReleaseNamespace.rules
-                                || !scope.toReleaseNamespace.rules.ruleItems
-                                || !scope.toReleaseNamespace.rules.ruleItems.length) {
-
-                                scope.toReleaseNamespace.viewType = 'rule';
-                                AppUtil.showModal('#grayReleaseWithoutRulesTips');
+                    function (result) {
```

Comment:
please keep the origin format to avoid unnecessary changes

Reply:
ok,I will do it.Thank you